doc/continuous-integration.md

```diff
@@ -0,0 +1,33 @@
+# Continuous integration
+
+## Main
+
+The [govuk_smart_answers CI instance](https://ci-new.alphagov.co.uk/job/govuk_smart_answers/) is triggered by new commits on the `master` branch of the [GitHub repo](https://github.com/alphagov/smart-answers). It runs the [`jenkins.sh`](https://github.com/alphagov/smart-answers/blob/master/jenkins.sh) script with the `RUN_REGRESSION_TESTS` not set. This script does the following:
+
+* Checks out the lastest [govuk-content-schemas](https://github.com/alphagov/govuk-content-schemas) which are used in a few of the tests
+* Installs the bundled gems
+* Runs govuk-lint-ruby (see [Rubocop docs](doc/rubocop.md))
```

Comment:
I don't think this link should include "doc/" as this continuous-integration.md file is already in the doc folder.